orix/tests/plot/test_unit_cell_plot.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018-2021 the orix developers
+#
+# This file is part of orix.
+#
+# orix is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# orix is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with orix.  If not, see <http://www.gnu.org/licenses/>.
+
+from packaging import version
+
+from diffpy.structure import Lattice, Structure
+from matplotlib import __version__ as _MPL_VERSION
+from matplotlib import pyplot as plt
+import numpy as np
+import pytest
+
+from orix.plot._util import Arrow3D
+from orix.quaternion import Orientation
+
+
+def test_unit_cell_plot_default():
+    ori = Orientation.random()
+    fig = ori.plot_unit_cell(return_figure=True)
+    assert len(fig.axes) == 1
+    axes = fig.axes[0]
+    assert len(axes.lines) == 12  # 12 edges in orthorhombic unit cell
+    # 6 Arrow3D -> 3 for both sample and crystal reference frames
+    if version.parse(_MPL_VERSION) >= version.parse("3.4"):  # pragma: no cover
+        assert len(axes.patches) == 6
+    else:
```

Comment:
I've got 3.5 on my machine, so these lines aren't covered

```suggestion
    else:  # pragma: no cover
```

Reply:
I run 3.5 too. There is a python 3.6 build on CI, which uses Matplotlib 3.3 I believe and is why this is needed.

Reply:
Yes, that's the [minimal version we support](https://github.com/pyxem/orix/blob/master/setup.py#L50).